### DISCUSSION TITLE
How do I repeat panel or rows for a list of object?

### DISCUSSION THREAD

**Discussion by dongxurr123** [grafana/grafana#58173] [Dashboards, Panels & Visualizations]
With this feature, I can define a list of object by custom variable, each object has the same field with others, so I can repeat for panels like this:

Variable's name: **_MyMonitors_** 
Variable's content:
_title:act_page,monitor_key:com.xxx.ddd.monitor.actPage;title:home_page,monitor_key:com.xxx.ddd.monitor.homePage_

Variables are seperated by comma, and fields in one variable are seperated by semicolon.

The 'panel title' set AS '**_${MyMonitors.title}_**'

The SQL in panel is like this:
```SQL
SELECT 
    create_time,
    monitor_num
FROM my_monitor_table
    WHERE monitor_key=${MyMonitors.monitor_key}
```

So when I repeat panel for variable "**_MyMonitors_**", there will be 2 panel created, one's title is "act_page", another's is "**_home_page_**",  and the SQL inspect shows one's SQL like this "**_monitor_key=com.xxx.ddd.monitor.actPage_**", another's SQL like "**_monitor_key=com.xxx.ddd.monitor.homePage_**"

**Comment by gelicia** [comment]
Hello, as you may have heard, we are [transitioning away from using discussions to discuss feature requests](https://github.com/grafana/grafana/discussions/73424). Due to the age and number of responses to this discussion, we are deciding to close it. If this is something you would like to see in Grafana, feel free to open an [issue](https://github.com/grafana/grafana/issues) so the discussion can continue. Thank you!
